presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -250,10 +253,37 @@ void PrestoServer::run() {
   connector::registerConnectorFactory(
       std::make_shared<connector::hive::HiveConnectorFactory>("iceberg"));
 
-  registerPrestoToVeloxConnector(
-      std::make_unique<HivePrestoToVeloxConnector>("hive"));
-  registerPrestoToVeloxConnector(
-      std::make_unique<HivePrestoToVeloxConnector>("hive-hadoop2"));
+  // Update connectorToCatalogInfo_ so the connector configs can be validated
+  // when the connectors are registered.
+  getCatalogConfigs();
+
+  // Update hiveConfigs map for validation when the connector is registered.
```

Comment:
I think what is confusing to me is that it says "update". But in fact, all that happens here is taking all the various catalog configurations from a per catalog lookup into a single vector to be passed into the connector constructor later on. Nothing is updated (changed), they are only collected into a single vector for easier processing.
Perhaps the comment and naming should reflect that.

Reply:
Thanks for the suggestion, updated the comment and variable names accordingly.